config/base/params.env

```diff
@@ -1 +1 @@
-odh-model-controller=quay.io/opendatahub/odh-model-controller:fast
+odh-model-controller=quay.io/spolti/odh-model-controller:RHOAIENG-6264
```

Comment:
probably leftover from testing

Reply:
yes, was intentional to make it easier to test.